image-filters/image-filters.markdown

```diff
@@ -1064,7 +1063,7 @@ public class ColorHelper {
 ```
 
 Clearly we can’t test this with whole images. Instead we can mock the images
```

Comment:
I think this should change - maybe to, "We don't want to test this with whole images, because we want images that we know the properties of and reason about. We approximate this by mocking the images and making them return an array of pixels..."